lib/validators/form_template_yaml_validator.rb

```diff
@@ -31,6 +31,8 @@ def validate(record)
         check_ids(record, field, existing_ids)
         check_descriptions_html(record, field)
       end
+
+      check_tag_groups(record, yaml.map { |f| f["tag_group"] }.compact.map(&:downcase).uniq)
```

Comment:
```suggestion
      check_tag_groups(record, yaml.map { |f| f["tag_group"] })
```

I feel like `.compact.map(&:downcase).uniq` should be done in `check_tag_groups`. Otherwise, it isn't clear that the `tag_group_names` argument expects an array of tag group names that have been compacted and downcased.

Reply:
good idea! resolved this.